Walking corpse syndrome

```diff
@@ -0,0 +1,10 @@
+The "Walkiing Corpse Syndrome" or Cotard's Delusion
```

Comment:
delete this file from the PR

Reply:
Can u please tell me how to delete this file. I tried doing it but i couldn't.